README.md

```diff
@@ -1 +1,44 @@
-# NYT-Article-clone
\ No newline at end of file
+# New-times-articleProject Name
+
+> This is a clone of the New York Times Article.
+
+![Screenshot]
+![Header](https://user-images.githubusercontent.com/63964646/112638309-2be6b580-8e50-11eb-9c78-e48fbfddcb83.png)
+![Footer](https://user-images.githubusercontent.com/63964646/112639150-ff7f6900-8e50-11eb-9499-f0d8ccbe64be.png)
+
+
+This is the first html-css project in the Microverse curriculum.
+
+## Built With
+
+- Html
+- Css
+
+## Live Demo
+
+[Live Demo Link](http://127.0.0.1:5502/index.html)
+
+
+## Author
+
+👤 **Ani Victor**
+
+- GitHub: [@chukwuemeka1234](@chukwuemeka1234)
+- Twitter: [@avc_victor](@avc_victor)
+- LinkedIn: [Ani chukwuemeka] (Ani chukwuemeka)
```

Comment:
All of these links, no one is working, all of them are broken, kindly fix them out before you merge this branch on main branch